### DISCUSSION TITLE
[V3] wire:model.live.throttle can lose last update made to a text input

### DISCUSSION THREAD

**Discussion by mc906** [livewire/livewire#7160] [1. Help]
### Livewire version

v3.0.10

### Which PHP version are you using?

PHP 8.1

### Issue
If you use `wire:model.live.throttle` on a text input, the last modification made to the input can be lost.

#### Getting there
This came up with the [example for an update-post component using a form object](https://livewire.laravel.com/docs/forms#extracting-a-form-object) from the v3 docs. I tried [live updating](https://livewire.laravel.com/docs/forms#live-updating-fields) with [throttling](https://livewire.laravel.com/docs/forms#throttling-input) for the inputs in that simple component in an isolated setting. The issue occurred both with real-time form saving using the `updated()` hook and saving on click of a submit button.

#### Observation
After typing out the title `Post 1245` into the input, the stored value is `Post 124`, cutting off the character typed last. The dirty state is removed shortly after typing the final character `5` but the stored value does not include it.

### Snippet of an updated input from the doc example
```
<input type="text" wire:model.live.throttle="form.title" wire:dirty.class="border-warning">
```

### Snippet of updated hook (`save()` as in the example from the docs)
```
[...]
use Livewire\Component;
use App\Livewire\Forms\PostForm;
use Livewire\Drawer\Utils;

class UpdatePost extends Component
{
    public PostForm $form;

    [...]

    public function updated($name, $value)
    {
        $this->form->fill([Utils::afterFirstDot($name) => $value]);
        $this->form->update();
    }

    [...]
}
```
### Questions
Is this working as expected? If so, how can I best make sure the last update to a text input is not lost in this scenario? Any help or comments would be very much appreciated. Thank you!

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by dorkster100** [comment]
after checking this out a bit deeper, although i don't think `throttle` should cause this issue, but probably the intention here is to use `debounce` instead which works correctly.

Writing for future reference as this pops up on google.

**Reply by mc906** [reply]
I agree that throttle should not cause this issue but `debounce` is what I ended up using instead. Thank you!
